### DISCUSSION TITLE
More pretty design for index page of docs

### DISCUSSION THREAD

**Discussion by nix6839** [raphamorim/rio#327] [General]
How about changing the design of the / page to something like https://vitepress.dev/? It looks cleaner, cooler, and more straightforward to me. That's just my personal opinion.

Also, what do you think about simplifying the logo? Keeping the design but removing the blurriness in the background and the glitches(?) in the frame might be nice. I believe that generally, users won't pay attention to such detailed aspects.

If you're interested, I could try working on it. :)

**Comment by raphamorim** [comment]
> How about changing the design of the / page to something like https://vitepress.dev/? It looks cleaner, cooler, and more straightforward to me. That's just my personal opinion.

That would be awesome @nix6839 , please feel free to go ahead and if you want to create a draft PR or small PRs and feel free to iterate/ping me anytime.

> Also, what do you think about simplifying the logo? Keeping the design but removing the blurriness in the background and the glitches(?) in the frame might be nice. I believe that generally, users won't pay attention to such detailed aspects.

Go for it, I think the logo as SVG got converted with these unfortunately details:

<img width="239" alt="Screenshot 2023-11-04 at 23 57 51" src="https://github.com/raphamorim/rio/assets/3630346/88dd3357-6c8a-424a-b8df-3b23b38df29e">

But the png version is much better in that aspect ( https://github.com/raphamorim/rio/blob/main/misc/osx/rio-logo.png )

<img width="358" alt="Screenshot 2023-11-04 at 23 58 48" src="https://github.com/raphamorim/rio/assets/3630346/219a0f01-14d7-405f-ba00-3c7224e0c69a">
